spec/models/labware_creators/multi_stamp_tubes_using_tube_rack_scan_spec.rb

```diff
@@ -229,7 +231,13 @@
     subject { LabwareCreators::MultiStampTubesUsingTubeRackScan.new(api, form_attributes) }
 
     it 'creates a plate!' do
+      # barcode from multi_stamp_tubes_using_tube_rack_scan/tube_rack_scan_valid.csv
+      subject.labware.barcode.prefix = 'AB'
+      subject.labware.barcode.number = '10000001'
```

Comment:
This feels a little hacky. Would it be possible and closer to reality to actually add the foreign barcode 'AB10000001' to `subject.labware`?